### PR TITLE
Updates to ASP.NET Identity quickstart docs

### PR DESCRIPTION
**Pull request by brockallen**
These docs use the new templates as their approach. 

@leastprivilege we can have a look next week

### CODE REVIEW

File: docs/quickstarts/6_aspnet_identity.rst [hunk 1]
```diff
@@ -2,180 +2,64 @@
...
-You will eventually delete the old project for IdentityServer (assuming you were following the other quickstarts), but there are several items that you will need to migrate over (or rewrite from scratch as described in the prior quickstarts).
+Install the IdentityServer Templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Start by creating a new "ASP.NET Core Web Application" project.
+If you haven't already, you will need to install the IdentityServer templates.
+They can be installed by running a ``dotnet`` command from the command line::
 
-.. image:: images/6_new_web_project.png
+    dotnet new -i IdentityServer.Templates
```
Comment (bartekmarnane) [inline docs/quickstarts/6_aspnet_identity.rst]: This should be: dotnet new -i IdentityServer4.Templates

Comment (brockallen) [reply docs/quickstarts/6_aspnet_identity.rst]: good catch, thanks!

**Comment by brockallen** [comment]
@leastprivilege given the feedback about the aspnet identity QS docs, perhaps we should merge this (it's been 8 months)

**Comment by leastprivilege** [comment]
OK for me. need to update to 2.1 though..

**Pull request closed without merging**